### PR TITLE
Misc fixes and donator items

### PR DESCRIPTION
**Pull request by francinum**
## About The Pull Request
Give shekel get fancy item.

## Why It's Good For The Game
About time I finished this feature.

## Changelog
:cl: kmc2000/francinum
server: Donation skinchanger/identification component
/:cl:

### CODE REVIEW

File: code/modules/clothing/spacesuits/plasmamen.dm [hunk 1]
```diff
@@ -190,52 +190,52 @@
...
 	icon_state = "rd_envirohelm"
 	item_state = "rd_envirohelm"
-	
+
 /obj/item/clothing/head/helmet/space/plasmaman/hop
 	name = "head of personnels envirosuit helmet"
 	desc = "An envirosuit helmet made for the Head of Personnel. Some corgi hair is stuck to it."
 	icon_state = "hop_envirohelm"
 	item_state = "hop_envirohelm"
-
```
Comment (Kmc2000) [inline code/modules/clothing/spacesuits/plasmamen.dm:231]: clrf?

File: code/game/turfs/simulated/floor/plating/misc_plating.dm [hunk 2]
```diff
@@ -195,7 +195,7 @@
 	icon_state = "smooth"
 	smooth = SMOOTH_MORE | SMOOTH_BORDER
 	canSmoothWith = list(/turf/open/floor/plating/ice/smooth, /turf/open/floor/plating/ice)
-	/turf/open/floor/plating/ice/colder
```
Comment (Kmc2000) [inline code/game/turfs/simulated/floor/plating/misc_plating.dm:198]: ??

File: nsv13/modules/donator/donator_component.dm [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+	owner_ckey = new_owner
+
+	RegisterSignal(parent, COMSIG_PARENT_EXAMINE, .proc/examine)
+
+/datum/component/donator/proc/examine(datum/source, mob/user, list/examine_list)
+	if(user.ckey == owner_ckey)
+		examine_list += "<span class='hypnophrase'>It's your special item.</span>"
+	else
+		examine_list += "<span class='hypnophrase'>It's a special item!</span>"
+		examine_list += "<span class='notice'>Consider donating to get one of these for yourself!</span>"
```
Comment (Kmc2000) [inline nsv13/modules/donator/donator_component.dm:17]: isn't it just . +=

File: code/game/turfs/simulated/floor/plating/misc_plating.dm (hunk 2 above)
Comment (Jalleo) [reply code/game/turfs/simulated/floor/plating/misc_plating.dm:198]: redundant see its the same line as below

File: nsv13/modules/donator/donator_component.dm (hunk 3 above)
Comment (Jalleo) [reply nsv13/modules/donator/donator_component.dm:17]: I think that can work but I am not certain.

**Pull request merged**